source/develop/release-management/features/network/ovirt-lldp-labeler.html.md

```diff
@@ -0,0 +1,156 @@
+---
+title: oVirt LLDP labeler
+category: feature
+authors: Ales Musil
+feature_name: oVirt LLDP labeler
+feature_modules: ovirt-lldp-labeler
+feature_status: In Development
+---
+
+# oVirt LLDP labeler
+
+## Summary
+
+oVirt LLDP labeler is a tool that utilizes [LLDP support](https://ovirt.org/blog/2017/11/webadmin-lldp/) 
+added to oVirt in 4.2. The tool helps to some extent automate network 
+configuration overhead in clusters that are managed by the labeler. 
+It can be used as a separate service or as one-time tool using its cli.
+
+### Owner
+
+- Feature Owner: Ales Musil (amusil)
+- E-mail: <amusil@redhat.com>
+
+
+### Benefit to oVirt
+
+The labeler is service that is capable of helping oVirt administrator
+to automate some network related tasks. According to configuration reported 
+via LLDP from peer switch, the labeler is capable of labeling host network 
+interface or bond interfaces together. 
+
+### Example VLAN flow
+
+Consider following scenario:
+
+Engine contains logical network `vlan10` tagged as VLAN with id 10
+and label `lldp_vlan_10`. Multiple hosts that have it's `eth1` 
+connected to switch with VLAN id 10 configured and LLDP reporting 
+enabled for the interfaces connected to the hosts. To automatically 
+connected our logical network `vlan10` with the `eth1`, on every host,
```

Comment:
connected our logical network `vlan10` with the `eth1` on every host,
(no comma)

Reply:
Done